### PR TITLE
🎨 1. part: invoice expiration fix (🗃️)

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
The invoice URL link we are getting from Payment Gateway has an expiration of 1 month. I plan to store the `stripe_invoice_id` that was already added to Payment Gateway and we are receiving it. Then use this ID to generate the link on the fly straight from Stripes. This way a valid link can be regenerated through our UI.

- 🗃️ adds `stripe_invoice_id` to the database
- ♻️ remove not needed payment_gateway_tax_feature_enabled if statement

Next PR: WIth this ID I will generate the invoice link on the fly from Stripes. 

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by GitHK — APPROVED** [review]
nice

File: packages/models-library/src/models_library/payments.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 from decimal import Decimal
-from typing import Any, ClassVar
+from typing import Any, ClassVar, TypeAlias
 
 from models_library.emails import LowerCaseEmailStr
 from pydantic import BaseModel, Field, validator
 
 from .products import StripePriceID, StripeTaxRateID
 
+StripeInvoiceID: TypeAlias = str
```
Comment (sanderegg) [inline packages/models-library/src/models_library/payments.py:9]: stupid question: why is this one not defined in the products like the above 2?

File: services/payments/src/simcore_service_payments/models/db.py [hunk 2]
```diff
@@ -23,6 +24,7 @@
     "wallet_id": 123,
     "comment": "This is a test comment.",
     "invoice_url": None,
+    "stripe_invoice_id": None,
```
Comment (sanderegg) [inline services/payments/src/simcore_service_payments/models/db.py:27]: I see you have a model of how it looks like after init, is there such a thing as well for once you get all the stripe stuff back?

**Review by sanderegg — APPROVED** [review]
love all these deleted things... ;) thanks!

**Review by sanderegg — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
love all these deleted things... ;)

File: packages/models-library/src/models_library/payments.py (hunk 1 above)
Comment (matusdrobuliak66) [reply packages/models-library/src/models_library/payments.py:9]: My rationale behind is that the StripePriceID and StripeTaxRateID are directly connected to the Product (they are even in the products_prices table in DB). While StripeInvoiceID is something purely connected to Stripe and Stripe client will live in the Payment service.

File: services/payments/src/simcore_service_payments/models/db.py (hunk 2 above)
Comment (matusdrobuliak66) [reply services/payments/src/simcore_service_payments/models/db.py:27]: There will be one in the next PR, as Stripe client will be introduced

**Review by pcrespov — APPROVED** [review]

File: packages/models-library/src/models_library/payments.py (hunk 1 above)
Comment (matusdrobuliak66) [reply packages/models-library/src/models_library/payments.py:9]: Ok, I will change in next pr that is coming

**Pull request merged**